service/src/test/java/uk/gov/hmcts/reform/fpl/service/CommonDirectionServiceTest.java

```diff
@@ -157,6 +158,23 @@ void constructDirectionForCCD_shouldConstructDirectionFromConfigurationAsExpecte
             .build());
     }
 
+    @Test
+    void numberDirections_shouldApplyCorrectNumberingWhenDirectionsAreMarkedAsRemoved() {
+        CaseData caseData = populateCaseDataWithSomeDirectionsMarkedAsRemoved().build();
+
+        List<Element<Direction>> directions = service.combineAllDirections(caseData);
+
+        List<String> numberedDirectionTypes = service.numberDirections(directions).stream()
+            .map(direction -> direction.getValue().getDirectionType())
+            .collect(toList());
+
+        List<String> expectedDirectionsTypes = IntStream.range(0, numberedDirectionTypes.size())
```

Comment:
```suggestion
        List<String> expectedDirectionTypes = IntStream.range(0, numberedDirectionTypes.size())
```

Reply:
this expected direction seems quite weak to me as well. I don't like logic to work out expected values as we're introducing potential failure points. Maybe this is fine however. I shall leave it with you

Reply:
I think I would prefer asserting the whole direction so I can see titles AND the removed things 🤔